Adding a new subchapter on rdm toolkits (#1562)
alan-turing-institute/the-turing-way: book/website/reproducible-research/rdm/rdm-toolkits.md

# RDM Toolkits
## Research data management toolkits

### What is a research data management toolkit?

A research data management toolkit provides information and advice about the data management activities in the whole data lifecycle. As such, it provides researchers with a strategy to manage their data to international standards.

Next to a website based around the research data management lifecycle, a research data management toolkit may also include "real world" use cases and scenarios. Additionally, a collection of tools to support the researcher may be available in a number of toolkits. 

### Why should you use a research data management toolkit?

A research data management toolkit will help you to maximize the efficiency and integrity of your research, and to increase your visibility and impact. Consulting a research data management toolkit will also learn you how to comply with requirements from funders, publishers and others. Last but not least, the toolkit can assist in finding and re-using datasets for your own research.

### How to choose a research data management toolkit?

> Information and materials are often sorted by topic and audience. Materials can be found relevant to your role (researcher, research support, IT specialist), your research domain (e.g.for life sciences: plant phenotyping, human data,...) or your problem (e.g. storage, data transfer).  

Reviewer: ```suggestion
Information and materials are often sorted by topic and audience. Materials can be found relevant to your role (researcher, research support, IT specialist), your research domain (for life sciences: plant phenotyping, human data,...) or your problem (such as storage, data transfer).  
```

A wide variety of research data management toolkits are available. Here, we list a number of toolkits:

* [JISC research data management toolkit](https://rdmtoolkit.jisc.ac.uk/rdm-for-researchers/)
* [ELIXIR research data management toolkit](https://rdm.elixir-europe.org/index.html)
* [University of New Hampshire data management toolkit](https://libraryguides.unh.edu/datamanagement)

> Notes
> *Many more toolkits are available: lots of them are however rather guides than toolkits. One could argue about the definition of a RDM toolkit*

Reviewer: ```suggestion
*Many more toolkits are available. Lots of them are rather guides than toolkits. One could argue about the definition of a RDM toolkit*
```
